friendshipper/src/routes/+layout.svelte

```diff
@@ -415,7 +356,7 @@
 				if ($appConfig.repoPath !== '') {
 					const [repoConfigResponse, repoStatusResponse, commitsResponse] = await Promise.all([
 						getRepoConfig(),
-						getRepoStatus(SkipDllCheck.False, AllowOfflineCommunication.False),
```

Comment:
Why do these parameters change?

Reply:
these are actually the function's defaults